addons/editor/stringtable.xml

```diff
@@ -60,6 +60,12 @@
             <Polish>Kontroluje czy wiadomości radiowe jednostki są odtwarzane przy postawieniu i wyznaczeniu punktu trasy, jedynie przy wyznaczeniu punktu trasy lub czy są wyłączone.</Polish>
             <Russian>Определяет будут ли ИИ подразделения сообщать по радио о своих действиях или нет.</Russian>
         </Key>
+        <Key ID="STR_ZEN_Editor_ParachuteSounds">
+            <English>Parachute Sounds</English>
+        </Key>
+        <Key ID="STR_ZEN_Editor_ParachuteSounds_Description">
+            <English>Controls whether jet flyby sounds are played when object positions are changed to a height where they are attached to parachutes.</English>
```

Comment:
```suggestion
            <English>Controls whether jet flyby sounds are played when an object is para-dropped via ALT drag.</English>
```

Reply:
Maybe "via the Raise Entity keybind." since the key can be rebound (I don't know why anyone would).

Reply:
I don’t know either but that could work.